Doc/library/os.rst

```diff
@@ -732,6 +764,45 @@ process and user.
       The function is now always available and is also available on Windows.
 
 
+.. function:: unshare(flags)
+
+   Disassociate parts of the process execution context, and move them into a
+   newly created namespace.
+   See the :manpage:`unshare(2)`
+   man page for more details.
+   The *flags* argument is a bit mask combining zero or more of the ``CLONE_*``
+   constants using ``|`` (bitwise or), that specifies which parts of the execution
+   context should be unshared from their existing associations and moved to a
+   new namespace..
+   If *flags* is specified as zero, no changes are made to the calling process's
+   execution context.
+
+   .. availability:: Linux 2.6.16 or newer.
+
+   .. versionadded:: 3.12
+
+   .. seealso::
+
+      The :func:`~os.setns` function.
+
+Flags to the :func:`unshare` function, if the implementation supports them.
+See the Linux manual for the exact effect and availability.
```

Comment:
```suggestion
See :manpage:`clone(2)` in the Linux manual
for their exact effect and availability.
```

Explicitly state and link the actual manual entry; I had to go digging to find it.